code/modules/food_and_drinks/recipes/drinks_recipes.dm

```diff
@@ -473,6 +473,13 @@
 	required_reagents = list(/datum/reagent/consumable/laughter = 1, /datum/reagent/consumable/cream = 1)
 	reaction_tags = REACTION_TAG_DRINK | REACTION_TAG_EASY | REACTION_TAG_OTHER
 
+/datum/chemical_reaction/drink/bureau
+	name = "Bureaucratic Perfection"
+	id = /datum/reagent/consumable/ethanol/bureau
+	results = list(/datum/reagent/consumable/ethanol/bureau = 3)
+	required_reagents = list(/datum/reagent/consumable/ethanol/gin = 1, /datum/reagent/consumable/ethanol/wine = 1, /datum/reagent/consumable/cream = 1)
```

Comment:
i'm curious why you chose these drinks specifically

Reply:
its supposed to be "fancy" seeming but sensible drinks all neatly organized in a cup